react-native/react/constants/platform.native.desktop.js

```diff
@@ -19,10 +19,20 @@ const envedPathOSX = {
   prod: 'Keybase'
 }
 
+function buildWin32SocketRoot() {
+  let appdata = process.env.APPDATA
+  // Remove leading drive letter e.g. C:
+  if (/^[a-zA-Z]:/.test(appdata)) {
+    appdata = appdata.slice(2)
+  }
+  return `\\\\.\\pipe\\kbservice\\${appdata}\\Keybase`
```

Comment:
keybase.exe and kbfs allow for an environment and/or command line variable to replace this. Also, currently it is different in develop vs. production.